### PR TITLE
Feat: Update cell related models

### PR DESCRIPTION
**Pull request by fedorgrab**
Updates:
* Add dataset_version_id to ingest info table
* Drop not null constraints from `obs_metadata_extra` and `original_cell_id` as for the current extract they are not necessary.
* Since in new data we now have "unknown" cell type instead of "naive cell" we have to handle this in ontology aware strategy

### CODE REVIEW

**Review by nmalfroy — APPROVED** [review]
lgtm!

File: src/casp/services/db/migrations/versions/t_2024-06-27_21-00-54_update_cell_related_models_before_new_.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+depends_on = None
+
+
+def upgrade() -> None:
+    op.alter_column("cells_cellinfo", "original_cell_id", existing_type=sa.VARCHAR(length=255), nullable=True)
+    op.alter_column(
+        "cells_cellinfo", "obs_metadata_extra", existing_type=postgresql.JSONB(astext_type=sa.Text()), nullable=True
+    )
+    op.add_column("cells_ingestinfo", sa.Column("dataset_version_id", sa.String(length=255), nullable=True))
+    # ### end Alembic commands ###
```
Comment (KevinCLydon) [inline src/casp/services/db/migrations/versions/t_2024-06-27_21-00-54_update_cell_related_models_before_new_.py:26]: This comment probably doesn't need to be here

**Review by KevinCLydon — APPROVED** [review]
Looks good to me!  I only have one minor comment and a completely unrelated question

File: requirements.txt [hunk 2]
```diff
@@ -26,7 +26,7 @@ matplotlib
 pandas
 google-cloud-run==0.7.1
 scikit-learn
-cellarium-ml @ git+https://github.com/cellarium-ai/cellarium-ml.git@0.0.4+module
+cellarium-ml @ git+https://github.com/cellarium-ai/cellarium-ml.git@0.0.7
```
[the remark this answers is not in the document]
Comment (fedorgrab) [reply requirements.txt]: That's a good point!

**Pull request merged**